src/components/Button.tsx

```diff
@@ -3,18 +3,20 @@ import React from "react";
 type Props = {
   title: string;
   onClick: (...args: any[]) => void;
+  className?: string;
 };
 
 const Button: React.FC<Props> = (props) => {
-  const { title, onClick } = props;
+  const { title, onClick, className } = props;
+
+  const defaultStyles =
+    "px-4 py-2 border-2 border-white bg-black rounded-xl cursor-pointer hover:scale-[1.2] transition-all duration-200 shadow-xl hover:bg-white hover:text-black hover:font-black";
+  const newClassnames = `${defaultStyles} ${className}`;
```

Comment:
this will show up as `` `${defaultStyles} undefined` `` if a className is not given. might be worth  declaring the default styles outside of the Button component as it is not necessary, and defaulting the className prop to an empty string, or use the classNames npm package. `const Button: React.FC<Props> = ({ title, onClick, className = "" }) => {`

Not really important though, ignore if not worth the effort

Reply:
Added default